common/test/TestDevice.cs

```diff
@@ -59,7 +59,7 @@ public static async Task<TestDevice> GetTestDeviceAsync(string namePrefix, TestD
 
         private static async Task<TestDevice> CreateDeviceAsync(TestDeviceType type, string prefix)
         {
-            string deviceName = prefix + Guid.NewGuid();
+            string deviceName = "E2E_" + prefix + Guid.NewGuid();
```

Comment:
you will need to modify other test classes that add this prefix explicitly, else, the test devices will now be called `E2E_E2E_MessageReceiveFaultInjectionTests_Sasl_dfcb23e9-f224-4fd1-8f70-7f55598d9941`.
This is in no way a blocker for tests, but instead just looks very very weird.

Reply:
Yes I know. I thought it should be ok to have a double prefix rather than fixing all files. We will be future proof though.

Reply:
I am looking at how many files though. I will be updating if they are just a few.